apsuite/commissioning_scripts/calc_coupling.py

```diff
@@ -0,0 +1,167 @@
+#!/usr/bin/env python-sirius
```

Comment:
She-bang lines are innocuous in python library modules, which are not supposed to be run by standalone executables. I think it should be dropped here.

Reply:
removed!